fedimintd/src/bin/distributedgen.rs

```diff
@@ -164,10 +157,7 @@ async fn main() {
             name,
             password,
         } => {
-            let salt: [u8; 16] = rand::random();
-            fs::write(dir_out_path.join(SALT_FILE), hex::encode(salt)).expect("write error");
-            let key = get_key(password, dir_out_path.join(SALT_FILE));
-            let config_str = gen_tls(&dir_out_path, p2p_url, api_url, name, &key);
+            let config_str = create_cert(dir_out_path, p2p_url, api_url, name, password);
             println!("{}", config_str);
```

Comment:
This is an exception where we actually want println right?

Reply:
I think so -- it's output of a CLI. If we `tracing::info` this, then it's harder to write bash scripts that take this as input.